### PR TITLE
world_iso list modified

### PR DESCRIPTION
**Pull request by HanaElattar**
# Closes # (if applicable).

## Checklist

- [ ] I tested my contribution locally and it seems to work fine.
- [x] Code and workflow changes are sufficiently documented.
- [x] Newly introduced dependencies are added to `envs/environment.yaml` and `envs/environment.docs.yaml`.
- [x] Changes in configuration options are added in all of `config.default.yaml`, `config.tutorial.yaml`, and `test/config.test1.yaml`.
- [x] Changes in configuration options are also documented in `doc/configtables/*.csv` and line references are adjusted in `doc/configuration.rst` and `doc/tutorial.rst`.
- [x] A note for the release notes `doc/release_notes.rst` is amended in the format of previous release notes, including reference to the requested PR.

### CODE REVIEW

File: scripts/config_osm_data.py [hunk 1]
```diff
@@ -161,25 +161,27 @@
...
+       "DZ": "algeria",
         "AO": "angola",
         "BJ": "benin",
         "BW": "botswana",
+        # "IO": "british-indian-ocean-territory", # Island
         "BF": "burkina-faso",
         "BI": "burundi",
         "CM": "cameroon",
-        # canary-islands    # Island
+       	// # canary-islands    # Island
```
Comment (davide-f) [inline scripts/config_osm_data.py]: Unnecessary "//"; this throws an error in the CI, which is a workflow that is run to check if the contributions are safe to merge

File: scripts/config_osm_data.py [hunk 2]
```diff
@@ -161,25 +161,27 @@
...
-        "CG": "congo-brazzaville",
+        # "KM": "comoros", # Island
+       	"CG": "congo-brazzaville",
         "CD": "congo-democratic-republic",
         "DJ": "djibouti",
         "EG": "egypt",
         "GQ": "equatorial-guinea",
         "ER": "eritrea",
         "ET": "ethiopia",
+	    "TF": "french-southern-territories",
```
Comment (davide-f) [inline scripts/config_osm_data.py]: Harmonize the data, there is here an unnecessary tab; this issue happens also below

File: scripts/config_osm_data.py [hunk 3]
```diff
@@ -161,25 +161,27 @@
 
 world_iso = {
     "Africa": {
-        "DZ": "algeria",
+       "DZ": "algeria",
```
Comment (davide-f) [inline scripts/config_osm_data.py]: Modify as it was before

**Review by davide-f — CHANGES_REQUESTED** [review]
Thank you for the contributions! Very detailed work :)
I added few simple comments.

Moreover, the pre-commit is not working.
To successfully use it, I recommend you to do as in https://pypsa-meets-africa.readthedocs.io/en/latest/installation.html , in particular:
- Use an IDE to edit python (recommended Visual Studio Code that is open)
- Install the pypsa-africa environment (see guide)
- Always make edits and commits after having enabled the environment [In VSCode, this may be easily done automatically by setting View->Command Pallette->Python Interpreter->pypsa-africa)

**Comment by davide-f** [comment]
@HanaElattar , if you need any help on the above, feel free to write on Discord or here :)

**Comment by HanaElattar** [comment]
Added a list of countries but couldn't test that the workflow runs. Could be for future work.

**Pull request merged**